sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -570,7 +570,7 @@ private[parquet] class ParquetFilters(
               }
 
               override def keep(value: Binary): Boolean = {
-                UTF8String.fromBytes(value.getBytes).startsWith(
+                value == null || UTF8String.fromBytes(value.getBytes).startsWith(
```

Comment:
Does that actually affect any end-to-end job in Spark?

Reply:
@dongjoon-hyun @HyukjinKwon 
It seem does not affect Spark end-to-end work currently. 
It seem difficult to add a test case in the current Parquet version.
So are we merging this PR or closing it and fixing it in the future?